### DISCUSSION TITLE
flakehell maintainer closed issue about whether that project is maintained

### DISCUSSION THREAD

**Discussion by mcarans** [flakeheaven/flakeheaven#15] [Governance]
https://github.com/flakehell/flakehell/issues/25

@pwoolvett @lyz-code @jarojasm95 

Note that he didn't reply in the issue and has not replied to my emails so it would be surprising if this marked the beginning of new activity to maintain flakehell.

**Comment by jose-rojas0** [comment]
seems like a couple PRs got merged, would be great to get a dialogue going though

**Comment by mcarans** [comment]
@jarojasm95 You are welcome to try. I had tried before through a GitHub issue and directly by email, but Sergey was unresponsive and there is still no feedback on the PR I had made.

**Comment by jose-rojas0** [comment]
How should we proceed?
https://github.com/flakehell/flakehell/issues/25#issuecomment-1010914370
